limacharlie/retention.go

```diff
@@ -0,0 +1,52 @@
+package limacharlie
+
+import (
+	"fmt"
+	"net/http"
+)
+
+type Stats struct {
+	Totals map[string]uint `json:"totals"`
+}
+type DetStats struct {
+	Totals map[string]map[string]int `json:"totals"`
+}
+
+func (org *Organization) OnlineStats(start string, end string) (Stats, error) {
```

Comment:
```suggestion
func (org *Organization) OnlineStats(start int64, end int64) (Stats, error) {
```

You can use `int64` for the timestamp, the underlying client will convert to string when sending to the API. Using `int64` makes it clearer that an epoch is expected.